src/components/DataTable.js

```diff
@@ -40,13 +40,25 @@ export default ajaxCaller(class DataTable extends Component {
   constructor(props) {
     super(props)
 
+    const { columnDefaults, entityType, entityMetadata } = props
+
+    const columnDefaultsObj = columnDefaults && JSON.parse(columnDefaults)
```

Comment:
If the workspace attribute is set to an invalid JSON string, the parsing can fail.